languages/python/oso/polar/polar.py

```diff
@@ -140,7 +140,8 @@ def load_str(self, string, filename=None):
                 try:
                     next(Query(query, host=self.host.copy()).run())
                 except StopIteration:
-                    raise PolarRuntimeException("Inline query in file failed.")
+                    source = query.source()
+                    raise PolarRuntimeException(f"Inline query failed: {source.get()}")
```

Comment:
Nit: it would be good to create an error type for this so that people can rescue it specifically without matching on the message, but that can be part of a (necessary) broader Python error handling cleanup.